vm/array_test.go

```diff
@@ -424,12 +424,14 @@ func TestCountMethodFail(t *testing.T) {
 
 	for _, tt := range testsFail {
 		evaluated := testEval(t, tt.input)
+
 		err, ok := evaluated.(*Error)
 		if !ok {
```

Comment:
I think this can be shorten like:
```go
if !ok || err.Class.ReturnName() != ArgumentError {
    t.Errorf("Expect ArgumentError. got=%T (%+v)", err, err)
}

Since we basically return similar fail message, this will be simpler to write.
However, this is just a thought not change request.